html2wp/setup.php

```diff
@@ -131,9 +136,19 @@ function html2wp_register_required_plugins() {
 			'source'           => 'https://github.com/html2wp/simple-live-editor/archive/master.zip',
 			'required'         => true,
 			'force_activation' => true,
-		),
+		)
 
 	);
 
+    if ( isset($html2wp_settings["forms"]) && !empty($html2wp_settings["forms"]) ) {
```

Comment:
Please revise https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage and https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#single-and-double-quotes